server/modules/activityLog/activityLogger.js

```diff
@@ -38,6 +38,10 @@ const type = {
   nodeCreate: 'nodeCreate',
   nodeValueUpdate: 'nodeValueUpdate',
   nodeDelete: 'nodeDelete',
+
+  // user
```

Comment:
why two different activities? shouldn't it be always userInvite? - when user join the application for the first time, then adding a userJoined activity log? what do you think?

Reply:
ok